Heimdall/Ethereum/NonceProvider.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  NonceProvider.swift
+//  Heimdall
+//
+//  Created by Luis Reisewitz on 30.10.17.
+//  Copyright © 2017 Gnosis. All rights reserved.
+//
+
+class NonceProvider {
+    func nextNonce() -> UInt {
+        // FIXME: dummy nonce, need to increment and get from blockchain maybe?
+        return 12
```

Comment:
Hmm should that be comited like that?

Reply:
Haven't really thought about this completely yet, so I would say yes.